### PR TITLE
Added ISO/IEC 8859-15 charset

### PR DESCRIPTION
**Pull request by TheCBaH**
This adds support for ISO/IEC 8859-15 charset .

### CODE REVIEW

File: src/encoding.ml [hunk 1]
```diff
@@ -275,3 +275,45 @@ let ebcdic_37_table = [|
...
+    0x0050; 0x0051; 0x0052; 0x0053; 0x0054; 0x0055; 0x0056; 0x0057;
+    0x0058; 0x0059; 0x005A; 0x005B; 0x005C; 0x005D; 0x005E; 0x005F;
+    0x0060; 0x0061; 0x0062; 0x0063; 0x0064; 0x0065; 0x0066; 0x0067;
+    0x0068; 0x0069; 0x006A; 0x006B; 0x006C; 0x006D; 0x006E; 0x006F;
+    0x0070; 0x0071; 0x0072; 0x0073; 0x0074; 0x0075; 0x0076; 0x0077;
+    0x0078; 0x0079; 0x007A; 0x007B; 0x007C; 0x007D; 0x007E; 0x007F;
+    (* ISO-8859-1 *)
+    0x0080; 0x0081; 0x0082; 0x0083; 0x0084; 0x0085; 0x0086; 0x0087;
+    0x0088; 0x0089; 0x008A; 0x008B; 0x008C; 0x008D; 0x008E; 0x008F;
+    0x0090; 0x0081; 0x0092; 0x0093; 0x0094; 0x0095; 0x0096; 0x0097;
```
Comment (aantron) [inline src/encoding.ml]: I think some of the `8x` code points should be `9x` on this line and the next, if I'm not mistaken.

File: test/test_encoding.ml [hunk 2]
```diff
@@ -83,6 +83,13 @@ let tests = [
     next_option chars ok (assert_equal None);
     next_option chars ok (assert_equal None));
 
+  ("encoding.iso_8859_1" >:: fun _ ->
```
Comment (aantron) [inline test/test_encoding.ml]: Should this have been a test for `iso_8859_15` instead? It looks like an exact duplicate of the `iso_8859_1` test right now.

**Comment by TheCBaH** [comment]
Thank you for spotting these embarrassing typos, they should be solved now.

**Comment by aantron** [comment]
No worries, we all benefit from review, me included :) Thanks for the PR :)

**Comment by aantron** [comment]
Would you like a speedy release to opam?

**Comment by TheCBaH** [comment]
> Would you like a speedy release to opam?

Nope, this is unnecessary.

**Pull request merged**